src/screening/nutrition/NutritionSummary.jsx

```diff
@@ -43,7 +40,6 @@ class NutritionSummary extends React.PureComponent {
         <ObsHistory
           concepts={[CONCEPTS.Weight, CONCEPTS.Height, CONCEPTS.BMI, CONCEPTS.Pregnant, CONCEPTS.MUAC]}
           editableEncounterTypes={[ENCOUNTER_TYPES.NutritionEncounterType]}
-          loading={this.state.loading}
```

Comment:
@mogoodrich , I had to remove the loading circle from here, because we are now updating the patient store more frequently, and each time the patient store is updated the loader appears, it was a bad user experience.